tests/integration/targets/user_grants/tasks/main.yaml

```diff
@@ -71,7 +71,7 @@
     - name: Assert grants updated
       assert:
         that:
-          - no_changes.changed == False
+          - no_changes.changed == True
```

Comment:
The test step this assertion references doesn't seem to change anything, so we might need to investigate what is causing the diff here :+1:

Reply:
actions is consider as "Updated grants" in both Update and No changes 
 and the changed status will have the value 'true" 
 we can have verify in other way like below 
       assert:
        that:
          - no_changes.grants.global.add_linodes == update_grants.grants.global.add_linodes